.github/workflows/spell-checker.yml

```diff
@@ -0,0 +1,33 @@
+name: Spell Checker
+on:
+  push:
+    branches:
+      - 'main'
+      - '[0-9]+.[1-9][0-9]*.x'
+    paths:
+      - 'docs/**'
```

Comment:
Don't we need to run this spell checker for all the files?

Reply:
Yes we can run it for all the files.